test/units/TEST-64-UDEV-STORAGE.sh

```diff
@@ -717,6 +717,11 @@ testcase_btrfs_basic() {
         /dev/disk/by-id/scsi-0systemd_foobar_deadbeefbtrfs{0..3}
     )
 
+    if ! modinfo btrfs; then
```

Comment:
IIRC, we have tested that before starting the test, see host_has_btrfs() in test-functions. Why the check does not work??

Reply:
These tests run in the mkosi based images, which do not reuse any of the logic from `test-functions`. Because the mkosi based images might be very different from the host, we can't check if the host has btrfs to determine whether the image will have btrfs so we skip inside the test instead of outside the test.